config/core/test-configs.yaml

```diff
@@ -2327,6 +2327,21 @@ test_configs:
   - device_type: r8a774a1-hihope-rzg2m-ex
     test_plans:
       - baseline
+      - baseline-nfs
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-lkdtm
+      - kselftest-seccomp
+      - ltp-crypto
```

Comment:
This one as well as `ltp-ima` don't appear to be run on staging except for one x86 Chromebook.

Reply:
?

Reply:
working ltp-ima https://lava.ciplatform.org/scheduler/job/509671

Reply:
p.s. you commented on ltp-crypto but wrote about ltp-ima

Reply:
ltp-crypto also work https://lava.ciplatform.org/scheduler/job/510077

Reply:
OK: https://staging.kernelci.org/test/plan/id/617a682c3f14e8a20aa6c442/

Yes I meant ltp-crypto as well as ltp-ima.  But it loosk like it's all working now.